src/scss/fluid-bootstrap/_breadcrumb.scss

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2011-2018, Hortonworks Inc.  All rights reserved.
+ * Except as expressly permitted in a written agreement between you
+ * or your company and Hortonworks, Inc, any use, reproduction,
+ * modification, redistribution, sharing, lending or other exploitation
+ * of all or any part of the contents of this file is strictly prohibited.
+ */
+
+.breadcrumb-item {
```

Comment:
I think we should use some sort of namespace for components

Reply:
I disagree. We want to create a theme for Bootstrap that can relatively easily be applied to existing projects that use Bootstrap. We also don't want people who know Bootstrap to have to learn a different set of classes. If we namespace things, we are not serving either of these goals.

Reply:
Sorry. My bad! I was under the wrong impression that *breadcrumb-item* was nested under de the *breadcrumb* component or something like that. Please ignore this.